src/DIRAC/Core/Utilities/Os.py

```diff
@@ -147,6 +149,52 @@ def sourceEnv(timeout, cmdTuple, inputEnv=None):
     return result
 
 
-@deprecated("Will be removed in DIRAC 8.1", onlyOnce=True)
+@deprecated("Will be removed in DIRAC 9.0", onlyOnce=True)
 def which(executable):
     return shutil.which(executable)
+
+
+def findImage(continer_root="container/apptainer/alma9/"):
+    """Finds the image for the current platform
+    
+       This looks into location "${CVMFS_locations}/${container_root}/${platform}/"
+       and expects to find one of the following platforms:
+       - x86_64
```

Comment:
```suggestion

       - x86_64
```

Newline before bullet list